utils/check_self_hosted_runner.py

```diff
@@ -25,7 +25,7 @@ def get_runner_status(target_runners, token):
         fp.write(json.dumps(offline_runners))
 
     if len(offline_runners) > 0:
-        failed = "\n".join(offline_runners)
+        failed = "\n".join([x["name"] for x in offline_runners])
```

Comment:
We can join a d dictionary object. And all we want is to join the offline runners' name.